geode-core/src/main/java/org/apache/geode/distributed/internal/InternalConfigurationPersistenceService.java

```diff
@@ -948,6 +954,9 @@ public CacheConfig getCacheConfig(String group) {
     String xmlContent = configuration.getCacheXmlContent();
     // group existed, so we should create a blank one to start with
     if (xmlContent == null || xmlContent.isEmpty()) {
+      if (createNew) {
+        return new CacheConfig("1.0");
```

Comment:
this is because of this line in cache-1.0.xsd:
`attribute name="version" use="required" type="{http://geode.apache.org/schema/cache}versionType" fixed="1.0"`
We could make that version value a static field.

Reply:
ok.. could we then at least have it as a final static field, rather than an arbitrary string in the middle of the code somewhere..